modules/service/repository/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "base_directory" {
+  type        = string
+  description = "The directory in the repository containing opslevel.yml."
+  default     = null
+}
+
+variable "name" {
+  type        = string
+  description = "The name displayed in the UI for the service repository."
+  default     = null
```

Comment:
This is required i think

Reply:
This is optional in the actual [schema](https://github.com/OpsLevel/terraform-provider-opslevel/blob/main/opslevel/resource_opslevel_service_repository.go#L93-L96), but should it be required?

Reply:
I just checked the API schema and this is for the displayName which is optional